src/examples/solana_stake.ts

```diff
@@ -0,0 +1,97 @@
+
+async function stakeOperations(signer: KeyPairSigner) {
+  const ctx = (await Coinbase.apiClients.stake?.getStakingContext({ network_id: NetworkIdentifier.SolanaDevnet, address_id: signer.address.toString(), asset_id: Coinbase.assets.Sol, options: {} }))?.data.context;
+
+  console.log("Stakeable: ");
+  console.log(ctx?.stakeable_balance.amount);
+  console.log(ctx?.stakeable_balance.asset);
+  let ops: Array<StakingOperation> = new Array<StakingOperation>();
+
+  {
+    const amount = ctx?.stakeable_balance.amount ? BigInt(ctx?.stakeable_balance.amount.toString()) : 0;
+    if (amount > 1)
+      ops.push((await Coinbase.apiClients.stake!.buildStakingOperation({
+        action: "stake",
+        address_id: signer.address.toString(),
+        asset_id: Coinbase.assets.Sol,
+        network_id: NetworkIdentifier.SolanaDevnet,
+        options: { amount: "100000000" },
+      }))!.data!);
+  }
+
+  console.log("Unstakeable: ");
+  console.log(ctx?.unstakeable_balance.amount);
+  console.log(ctx?.unstakeable_balance.asset);
+
+  {
+    const amount = ctx?.unstakeable_balance.amount ? BigInt(ctx?.unstakeable_balance.amount.toString()) : 0;
+    if (amount > 1)
+      ops.push((await Coinbase.apiClients.stake!.buildStakingOperation({
+        action: "unstake",
+        address_id: signer.address.toString(),
+        asset_id: Coinbase.assets.Sol,
+        network_id: NetworkIdentifier.SolanaDevnet,
+        options: { amount: "100000000" },
+      }))!.data!);
+  }
+
+
+  console.log("Claimable: ");
+  console.log(ctx?.claimable_balance.amount);
+  console.log(ctx?.claimable_balance.asset);
+
+  {
+    const amount = ctx?.claimable_balance.amount ? BigInt(ctx?.claimable_balance.amount.toString()) : 0;
+    if (amount > 1)
+      ops.push((await Coinbase.apiClients.stake!.buildStakingOperation({
+        action: "claim_stake",
+        address_id: signer.address.toString(),
+        asset_id: Coinbase.assets.Sol,
+        network_id: NetworkIdentifier.SolanaDevnet,
+        options: { amount: "100000000" },
+      }))!.data!);
+  }
+
+
+  const rpc = createSolanaRpc("https://api.devnet.solana.com");
+
+  for (let op of ops) {
+    for (let tx of op.transactions) {
+      let txmsg = getTransactionDecoder().decode(bs58.default.decode(tx.unsigned_payload));
+      let signed = await signTransaction([signer.keyPair], txmsg);
+      let sig = await rpc.sendTransaction(getBase64EncodedWireTransaction(signed), { encoding: "base64" }).send();
+
+      console.log(`txLink: ${getTxLink(NetworkIdentifier.SolanaDevnet, sig)}`)
+    }
+  }
+
+}
+
+function getTxLink(networkID: string, signature: string): string {
+  if (networkID == "solana-mainnet") {
+    return "https://explorer.solana.com/tx/" + signature;
+  } else if (networkID == "solana-devnet") {
+    return "https://explorer.solana.com/tx/" + signature + "?cluster=devnet";
+  }
+
+  return ""
+}
+
+Coinbase.configureFromJson({ filePath: coinbaseApiKeyPath() });
+
+const kp = getKeypair();
```

Comment:
Can we use more descriptive/complete variable names in these examples across the board? AFAIK, it's more common to be descriptive in variable names in TS than Golang - we want to follow the SDK language's industry standards.